doc/faq.rst

```diff
@@ -20,7 +20,6 @@ sy-kit learn. sci stands for science!
 Why scikit?
 ------------
 There are multiple scikits, which are scientific toolboxes built around SciPy.
-You can find a list at `<https://scikits.appspot.com/scikits>`_.
```

Comment:
I'm okay with this change. There does not seem to be an alternative to the link: https://github.com/scipy/scipy.org/issues/382